stl/inc/xutility

```diff
@@ -2005,6 +2004,20 @@ _NODISCARD constexpr size_t size(const _Ty (&)[_Size]) noexcept {
     return _Size;
 }
 
+#if __HAS_CXX20
+template <class _Container>
+[[nodiscard]] constexpr auto ssize(const _Container& _Cont)
+    -> common_type_t<ptrdiff_t, make_signed_t<decltype(_Cont.size())>> {
+    using _rTy = common_type_t<ptrdiff_t, make_signed_t<decltype(_Cont.size())>>;
+    return static_cast<_rTy>(_Cont.size());
+}
+
+template <class _Ty, ptrdiff_t _Size>
+[[nodiscard]] constexpr ptrdiff_t ssize(const _Ty (&)[_Size]) {

```

Comment:
Missing `noexcept` depicted in the Standard.

Reply:
Done